src/planet-a/Satellites/SatPlanet.cpp

```diff
@@ -37,8 +37,11 @@ UCompPlanet& SatPlanet::add_planet(
         osp::universe::Universe& rUni, Satellite sat, double radius, float mass,
         float resolutionSurfaceMax, float resolutionScreenMax)
 {
+
+    float activateRadius = float(radius) * 256.0f;
+
     rUni.get_reg().emplace<UCompActivatable>(sat);
-    rUni.get_reg().emplace<UCompActivationRadius>(sat, float(radius));
+    rUni.get_reg().emplace<UCompActivationRadius>(sat, activateRadius);
```

Comment:
Is the downcast to float intentional for perf reasons?

I'm skeptical that its any faster than doing the calculation as double.